templates/validator/heading.html

```diff
@@ -1,59 +1,61 @@
 {{ define "validatorHeading" }}
-  <div class="my-3 py-2">
-    <div class="d-md-flex justify-content-md-between">
-      <div class="d-flex mb-1">
-        <h1 class="h4 mb-1 mb-md-0 d-flex align-items-center">
+  <div style="margin-top: 0.625rem;">
+    <div class="d-flex justify-content-between">
+      <div class="d-md-flex align-self-center">
+        <h1 class="style-table-title mb-0 mr-2 text-nowrap">
           <!-- using nonconventional characters for whitespaces because of linter -->
-          <span class="mr-2">Validator <span class="d-inline">{{ if and (ne .Status "deposited") (ne .Status "deposited_invalid") (ne .Status "deposited_valid") }}{{ .Index }}{{ end }}{{ if ne .Name "" }}&nbsp;({{ formatValidatorName .Name }}){{ end }}</span></span>
-          <span class="mx-1" data-toggle="tooltip" title="Copy public key to clipboard" data-clipboard-text="0x{{ printf "%x" .PublicKey }}">
-            <button class="btn btn-dark text-white btn-sm" type="button" id="copy-button">
-              <i class="fa fa-copy"></i>
-            </button>
-          </span>
-          <span class="mx-1" data-toggle="tooltip" title="Edit validator name">
-            <button class="btn btn-dark text-white btn-sm" type="button" id="edit-button" data-toggle="modal" data-target="#edit-validator-modal">
+          Validator{{ if and (ne .Status "deposited") (ne .Status "deposited_invalid") (ne .Status "deposited_valid") }} {{ .Index }}{{ end }}
+        </h1>
+        {{ if .Tags }}<div>{{ formatValidatorTags .Tags }}</div>{{ end }}
+        {{ if eq .Name "" }}
```

Comment:
We only show the edit button if the validator does not have a name yet, is that intentional?
In Figma it is always visible.

Reply:
Yes this is intentional. The name can still be changed in the "More" menu.